services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/mapper/StaticTableSchema.java

```diff
@@ -282,18 +282,40 @@ private Builder(Class<T> itemClass) {
         }
 
         /**
-         * A higher-precedence {@link AttributeConverterProvider} than the default one provided by the table schema.
-         * The {@link AttributeConverterProvider} must provide {@link AttributeConverter}s for all types used in the schema.
+         * Specifies the {@link AttributeConverterProvider}s to use with the table schema.
+         * The list of attribute converter providers must provide {@link AttributeConverter}s for all types used
+         * in the schema. The attribute converter providers will be loaded in the strict order they are supplied here.
          * <p>
-         * The table schema has a default, internal, AttributeConverterProvider which provides standard converters
-         * for most primitive and common Java types. Use custom AttributeConverterProvider when you have specific
-         * needs for type conversion that the defaults do not cover.
+         * Calling this method will override the default attribute converter provider
+         * {@link DefaultAttributeConverterProvider}, which provides standard converters
+         * for most primitive and common Java types, so that provider must included in the supplied list if it is to be
+         * used. Providing an empty list here will cause no providers to get loaded.
```

Comment:
That'd be not using the annotation at all. Right?!

Reply:
StaticTableSchema should not use the annotation. I was thinking the defaultProvider() method on the base interface if it's still there, otherwise DefaultAttributeConverterProvider.create() or whatever.

Reply:
Sorry, I was still in the bean space. Yes.